tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -53,7 +53,7 @@ else
         fi
         docker build -t "onnxruntime-$IMAGE" --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} -f $DOCKER_FILE .
     elif [ $BUILD_DEVICE = "tensorrt" ]; then
-        IMAGE="ubuntu16.04-cuda10.0-cudnn7.4-tensorrt5.0"
+        IMAGE="ubuntu18.04-cuda10.1-cudnn7.6-tensorrt6.0"
```

Comment:
run_dockerbuild.sh gets invoked from the various linux yaml files with -o option representing the BUILD_OS
ideally the image name should be $BUILD_OS-$CUDA_VER rather than hardcoded as it is for all the different options here.
then in the tensorrt yaml file we could pass in ubuntu18.04 instead of ubuntu16.04
and also use the -c option to specify cuda version